newjersey/navigator.business.nj.gov
# feat: [#188540268] Implement new login navigation pattern

## Description


Adds a new login page that checks for a user's email in our database before triggering a login via myNJ.

### Ticket



This pull request resolves [#188540268](https://www.pivotaltracker.com/story/show/188540268).

### Approach


Relatively straightforward. The main thing was making sure that all access points get updated from the old way, getting directed straight to myNJ login, to being directed to this intermediary page first.

### Steps to Test



Manually checking the flow:
1. Go through onboarding
2. Click "Log In" from nav bar
 3. Input a bad email for various error states:
	1. a string that isn't a properly formatted email
	2. an email that hasn't been linked to myNJ
4. Go to "Link with myNewJersey" from nav bar dropdown
5. Fill out and submit form, get redirected to profile page
6. Go back to "Log in"
7. This time, submitting that self-registered email should route you to myNJ to log in and successfully get redirected back to navigator upon completion

## Code author checklist
